src/azure-cli/azure/cli/command_modules/monitor/commands.py

```diff
@@ -244,3 +250,6 @@ def load_command_table(self, _):
         g.command('list', 'list_intelligence_packs')
         g.command('enable', 'enable_intelligence_pack')
         g.command('disable', 'disable_intelligence_pack')
+
+    with self.command_group('monitor', metric_alert_sdk, custom_command_type=monitor_general_custom) as g:
+        g.custom_command('clone', 'clone_existed_settings')
```

Comment:
Shouldn't we mark this command as preview?

Reply:
Sure